### PR TITLE
Add Rack::Protection

### PR DESCRIPTION
**Pull request by sethvargo**

### CODE REVIEW

File: website/Gemfile.lock [hunk 1]
```diff
@@ -80,18 +81,18 @@ GEM
...
       execjs (~> 2.0)
       haml (>= 4.0.5)
       kramdown (~> 1.2)
-      middleman-core (= 3.3.12)
+      middleman-core (= 3.3.13)
       middleman-sprockets (>= 3.1.2)
       sass (>= 3.4.0, < 4.0)
       uglifier (~> 2.5)
-    middleman-core (3.3.12)
+    middleman-core (3.3.13)
```
Comment (catsby) [inline website/Gemfile.lock]: @sethvargo this version was yanked, did you make this PR from `master`?

Comment (sethvargo) [inline website/Gemfile.lock]: Weird - that looks like a Rubygems issue, since it shouldn't be including that in the resolver output. Let me see....

**Comment by sethvargo** [comment]
@catsby fixed

**Comment by catsby** [comment]
LGTM

**Pull request merged**